Makefile

```diff
@@ -273,7 +273,7 @@ run-android: export TARGET := android
 run-android: ##@run Build Android APK and start it on the device
 	npx react-native run-android --appIdSuffix debug
 
-SIMULATOR=iPhone 13
+SIMULATOR=iPhone 14
```

Comment:
I upgraded to the latest macOS and latest XCode and now my system complains that I don't have iPhone 13, so I made a local change to Makefile and forgot to remove it) 
Btw, what iPhone version do you use on the simulator? Wondering if a lot of us have similar local changes locally, maybe we can make it to a Makefile.

Reply:
we all should use `iPhone 11 Pro` because it has the same size as Figma designs but I usually use `make run-ios SIMULATOR="iPhone 11 Pro"` btw you can always add any version of simulator, im wondering how this SIMULATOR value is used ?

Reply:
It is used to set the version if it is not provided directly, I believe:
```
ifneq ("$(SIMULATOR)", "")
	npx react-native run-ios --simulator="$(SIMULATOR)"
else
```